src/CreepLaw/DiffusionCreep.jl

```diff
@@ -232,7 +232,9 @@ returns the derivative of strainrate versus stress
     @unpack_val n, r, p, A, E, V, R = a
     FT, FE = a.FT, a.FE
 
-    return fastpow(FT * TauII, -1 + n) *
+    # computed symbolically
```

Comment:
Could you test if computing this derivative with `ForwardDiff.jl` is actually faster than the symbolic derivative?